e2e/terraform/provision-nomad/main.tf

```diff
@@ -105,11 +99,10 @@ resource "null_resource" "upload_nomad_binary" {
   }
 }
 
-resource "null_resource" "upload_custom_configs" {
```

Comment:
I removed `upload_custom_configs`, as it's superseded by uploading all of config, including `custom` profile.  Though, I'm not sure if we special case `custom` profile somehow.

Reply:
Not necessarily a blocker, but at least for consideration:

* If you're uploading the configuration on every `terraform apply`, there's no particular reason to have a `custom` profile at all. People can just change the defaults. That's how we had it before and I agree it does seem like that was more intuitive for folks.
* If you unconditionally upload the configuration, there's no way for the services to know that it changed on update. It might be worth adding logic to the `local-exec` invocation here that tests whether Nomad/Consul/Vault are running and restart them if so.